wfsim/load_resource.py

```diff
@@ -259,12 +268,13 @@ def __init__(self, config=None):
             # Garfield luminescence timing samples
             # if config.get('s2_luminescence_model', False) == 'garfield':
             if 'garfield' in config.get('s2_luminescence_model', ''):
-                s2_luminescence_map = straxen.get_resource(files['s2_luminescence'], fmt='npy_pickle')['arr_0']
-                # Get directly the map for the simulated level
-                liquid_level_available = np.unique(s2_luminescence_map['ll'])  # available levels (cm)
-                liquid_level = config['gate_to_anode_distance'] - config['elr_gas_gap_length']  # cm
-                liquid_level = min(liquid_level_available, key=lambda x: abs(x - liquid_level))
-                self.s2_luminescence = s2_luminescence_map[s2_luminescence_map['ll'] == liquid_level]
+                s2_luminescence_map = straxen.get_resource(files['s2_luminescence'], fmt='npy')
+                self.s2_luminescence = s2_luminescence_map
+            
+            if 'garfield_gas_gap' in config.get('s2_luminescence_model', ''):
```

Comment:
I would put this if condition first, and go with `elif 'garfield' in  config...` because when you have `'garfield_gas_gap'` both conditions are correct

Reply:
Sure, I agree